twisted/pydoctor
# Add pydoctor help directive.

Scope
=====

It adds a sphinx directive for producing the help output.

it also updates the tox docs env to fail on Sphinx warnings and the run is integrated into the Static CI build. 

With this change, we no longer have Sphinx warnings... all clean :)

There are no unit tests for the directive... but this should be private.


How to test
=========

RTD page https://pydoctor--253.org.readthedocs.build/en/253/help.html  ... you might see an old cache.

Local build
```
tox -e docs
firefox build/docs/help.html
```
